src/shared/hooks/useFilterConditionalLogicByItem.ts

```diff
@@ -0,0 +1,31 @@
+import { ConditionalLogic } from 'shared/state';
+import { getEntityKey } from 'shared/utils';
+import { getItemConditionDependencies } from 'modules/Builder/features/ActivityItems/ActivityItems.utils';
+import { ItemFormValues } from 'modules/Builder/types';
+import { useCurrentActivity } from 'modules/Builder/hooks';
+import { useFormContext } from 'react-hook-form';
+
```

Comment:
pls move to builder/hooks and fix spaces

Reply:
done